app/src/test/java/com/app/calculator/OperationPresenterTest.java

```diff
@@ -10,4 +10,18 @@
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 public class OperationPresenterTest {
```

Comment:
This class is not testing any method of the presenter, the asserts found here are not telling you that the presenter works correctly. Consider testing the methods on the presenter and if needed a refactor of the class so is able to be tested.